framework_tvb/tvb/adapters/analyzers/bct_adapters.py

```diff
@@ -246,21 +256,23 @@ def launch(self, connectivity, **kwargs):
 class DistanceRDA(DistanceRDM):
     """
     """
-    _ui_name = "Reachability and distance matrices (Algebraic path count)"
+    _ui_name = "Reachability and distance matrices (Algebraic path count): Connection matrix"
     _ui_description = bct_description("reachdist.m")
     _matlab_code = "[R,D] = reachdist(A);"
 
 
 class DistanceNETW(DistanceDBIN):
     """
     """
-    _ui_name = "Network walks"
+    _ui_name = "Network walks: Connection Matrix"
```

Comment:
Whenever we have just `Connection Matrix` at the end, I think it is safe to not put it there at all.

Reply:
reviewed all naming